shared-libs/rules-engine/src/rules-emitter.js

```diff
@@ -51,6 +55,9 @@ module.exports = {
         },
       });
     } catch (err) {
+      if (feedbackService) {
```

Comment:
What's the case where this is missing? I'd prefer that we can ensure it is always present and remove the conditional. Even if it is a legitimate case that it not be injected, my preference is still to ensure its presence via the null object pattern where up in the constructor we'd set the null object as a default like this:

```
  initialize: (settings, Feedback = new NullFeedbackService()) => {
```